src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3213,9 +3213,12 @@ def __init__(self, name=None, version=None, is_preview=False, supported_func_ver
 
         # used for displaying stacks
         def to_dict(self):
-            return {"runtime": self.name,
-                    "version": self.version,
-                    "supported_functions_versions": self.supported_func_versions}
+            d = {"runtime": self.name,
+                 "version": self.version,
+                 "supported_functions_versions": self.supported_func_versions}
+            if self.linux:
+                d["linux_fx_version"] = self.site_config_dict.linux_fx_version
```

Comment:
@StrawnSC this will display the linuxFxVersion with | which is probably what is supported by the runtime, but in webapps list-runtimes we convert this to : delimiter to help with | being special behavior with PS. I think this is OK (esp since functionapp create takes lang & runtime as 2 separate inputs)but could you share the new output with the function apps PM as well make sure this is what they expect as well. Thanks!

Reply:
Sounds good. Are there PMs in particular I should share this with besides Thiago?

Reply:
Since he is the one requesting this change - I would start with him. Thanks!

Reply:
@panchagnula @StrawnSC Any update? May I ask if this PR needs to be released in this sprint?

Reply:
@zhoxing-ms thanks for checking. this is good to go on our end. got confirmation from PM on our end as well. This sprint release please.